docs/src/components/PageSearch/PageSearch.js

```diff
@@ -66,6 +66,10 @@ class PageSearch extends Component<Props, State> {
     this.state = { value: '', suggestions: [] };
   }
 
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyDown);
```

Comment:
we need to remove it in `componentWillUnmount `

Reply:
Good shout!